static/app/components/workflowEngine/gridCell/actionCell.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+
+import {IconCircledNumber} from 'sentry/components/iconCircledNumber';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconMail} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+
+const ActionMetadata = {
+  slack: {name: t('Slack'), icon: <PluginIcon pluginId="slack" size={20} />},
+  discord: {name: t('Discord'), icon: <PluginIcon pluginId={'discord'} size={20} />},
+  email: {name: t('Email'), icon: <IconMail size="md" />},
+};
+
+export type Action = keyof typeof ActionMetadata;
+
+type ActionCellProps = {
+  actions: Action[];
+};
+
+export function ActionCell({actions}: ActionCellProps) {
+  if (!actions || actions.length === 0) {
+    return <div>{t('No actions')}</div>;
+  }
+  if (actions.length === 1 && actions[0]) {
+    return (
+      <Inline>
+        <IconContainer>{ActionMetadata[actions[0]].icon}</IconContainer>
+        {ActionMetadata[actions[0]].name}
+      </Inline>
+    );
+  }
+  const actionsList = actions
+    .map(action => ActionMetadata[action].name)
+    .reduce((acc, action) => `${acc}, ${action}`);
```

Comment:
Equivalent but easier to read than `reduce`. Could also probably be inlined in the markup just as easily.

```suggestion
    .map(action => ActionMetadata[action].name)
    .join(', ');
```